embedded-graphics/src/draw_target.rs

```diff
@@ -100,8 +107,18 @@ use crate::{
 ///
 /// In addition to defining [`draw_pixel`], an implementation of [`DrawTarget`] can also provide
 /// alternative implementations for hardware accelerated drawing operations. This example implements
-/// `DrawTarget` for a display that supports drawing hardware accelerated styled [`Rectangle`]s. The
-/// `draw_rectangle()` method overrides the default implementation.
+/// `DrawTarget` for a display without a framebuffer that supports hardware accelerated drawing of
+/// styled [`Rectangle`]s.
+///
+/// The default implementation of `draw_rectangle()` (as well as `draw_line()`, `draw_circle()` and
+/// `draw_triangle()`) defer to `draw_iter()` internally. In this example, the default
+/// implementation of `draw_rectangle()` is overridden to allow usage of accelerated draw commands
+/// specific to the targeted hardware.
+///
+/// As this example doesn't use a framebuffer, a `flush()` method is not required. All draw
```

Comment:
The way the framebuffer is flushed and the name of the method isn't relevant here. The previous example uses the more generic `"flush" operation` which could also be used here.

Reply:
Tweaked as well, thanks for the suggestion